pkg/reconcile/reconcilers.go

```diff
@@ -490,6 +493,13 @@ func newDeployment(reaper *api.Reaper, cassDcService string) *appsv1.Deployment
 		}
 	}
 
+	initContainerEnvVar := make([]corev1.EnvVar, len(envVars))
```

Comment:
Shouldn't this be len(envVars) + 1 ? Instead of letting append later on to grow the size again.